pulsar/internal/backoff.go

```diff
@@ -40,6 +43,7 @@ func (b *Backoff) Next() time.Duration {
 	} else if b.backoff.Nanoseconds() > maxBackoff.Nanoseconds() {
 		b.backoff = maxBackoff
 	}
+	jitter := rand.Float64() * float64(b.backoff) * jitterPercentage
```

Comment:
Does the generator need to be seeded (https://golang.org/pkg/math/rand/#Seed)?

Reply:
Yes, you are right. I updated the PR.